src/expandableCalendar/week.tsx

```diff
@@ -17,6 +17,13 @@ export type WeekProps = CalendarProps & {
   context?: CalendarContextProps;
 };
 
+function arePropsEqual(prevProps: WeekProps, nextProps: WeekProps) {
+  const {context: prevContext, markedDates: prevMarkings, ...prevRest} = prevProps;
+  const {context: nextContext, markedDates: nextMarkings, ...nextRest} = nextProps;
+
+  return isEqual(prevContext, nextContext) && isEqual(prevMarkings, nextMarkings) && isEqual(prevRest, nextRest);
```

Comment:
Yes. That's good. Just a rename as we use 'others' and not 'rest', so `prevOthers` and `nextOthers` or simply `prevProps` and `nextProps`.